### ISSUE TITLE
Save zip without compression

### ISSUE THREAD

**Issue by GDenisC** [Ne-Lexa/php-zip#94]
| Q                            | A
| -----------------------------| ---
| Library version(s) affected: | 4.01
| PHP version(s):              | 8.1.2
| OS (with bit depth):         | Windows 10 64-bit

**Description**  
How save zip without compression (all files)?

**How to reproduce**  
```php
$zip = new ZipFile();
$zip->openFile($fileName);
/* some code */
$zip->setCompressionMethodEntry(what do you mean with entry?, ZipCompressionMethod::STORED)->/* etc */;
```

**Comment by odan** [comment]
According to the documentation:

```php
$zip->addFile($file, $entryName, \PhpZip\Constants\ZipCompressionMethod::STORED); // No compression

or

$zip->addFromString('file', 'content', \PhpZip\Constants\ZipCompressionMethod::STORED);
```

> what do you mean with entry?

The $entryName is the filename within the zip archive.

**Comment by GDenisC** [comment]
i mean **all files** (with directories), because i need `$zip->openFile($fileName);`

**Comment by odan** [comment]
The `openFile` method is to open a ZIP file and can be used in combination with the `extractTo` method to extract files from that ZIP file.

Are you trying to "change" an existing ZIP file to an ZIP file with uncompressed content?
Can please add more details about your specific use-case?

**Comment by GDenisC** [comment]
Yes, I want to open a zip file with a lot of compressed files, export it, change some files and save as uncompressed zip.

**Comment by odan** [comment]
You can already do this with the existing methods.
